.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,24 +1,21 @@
-<!--
-  - Thanks for submitting code to Oppia! Please fill out the following as part of
-  - your pull request so we can review your code more easily.
-  -->
-
 ## Explanation
-<!--
-  - Explain what your PR does. If this PR fixes an existing bug, please include
-  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
-  - when this PR is merged.
-  -->
 
-## Checklist
-- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
-- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
+This PR fixes #[fill_in_number_here].
+
+This PR does the following and why...
```

Comment:
This PR does the following: [Explain what your PR does and why]

(and drop the "delete" line below)

Reply:
done!